package.json

```diff
@@ -37,10 +37,16 @@
   ],
   "dependencies": {
     "cheerio": "^0.19.0",
-    "got": "^3.0.0",
+    "got": "^4.2.0",
     "meow": "^3.3.0"
   },
   "devDependencies": {
-    "ava": "0.0.4"
+    "ava": "*",
+    "xo": "*"
+  },
+  "xo": {
+    "ignore": [
```

Comment:
Should be [`ignores`](https://github.com/sindresorhus/xo#ignores) according to the docs. Out of my own experience, `ignore` works as well but I think we should be consistent in this one :).